### ISSUE TITLE
Unable to Debug

### ISSUE THREAD

**Issue by BenWinchester** [BouncingSquaresNew/HexaboxesRepo#2]
Not sure if it works, but I've set out a class structure :)
(It still doesn't actually do anything)

**Comment by ChristianLoizou** [comment]
How do you download it? What is the program i need called again?

**Comment by BenWinchester** [comment]
Hi,
(It looks like a lot of steps, but it's really not that complicated)

To install the Android Studio (the development tool) you need to go to
http://developer.android.com/sdk/index.html and run the installer. Once it's installed, run Android Studio :)

Then, on your phone, you need to go to
Settings -> Development Settings and enable USB debugging.
If you can't see Development settings, you'll need to activate it by going to Settings -> About Phone -> And click on build number 7 times (to enable developer options.

Then, you'll need to go to the Repo, and click on "Download Zip" in the bottom right at https://github.com/BouncingSquaresNew/HexaboxesRepo

You'll then need to open the project in the Android Studio (by clicking on "import project" or "open project", and navigating to the folder /Hexaboxes/workspace1 in the little popup window :)

Once the code has fired up, you'll see a green 'play' icon in the top bar: plug your phone into the computer, and click on it

The app will install and run :D

**Comment by ChristianLoizou** [comment]
Ok,  thanks.  I'll download it in lunch break today 😊

**Comment by BenWinchester** [comment]
Did it work? :smiley:

**Comment by ChristianLoizou** [comment]
No, i cant seem to find the import file button, but i have been able to run the sdk. It just comes up with Tools and Extras buttons...

**Comment by BenWinchester** [comment]
When it opens, you should get a dialogue like this: 
![image](https://cloud.githubusercontent.com/assets/8342509/5745369/20277cd0-9c20-11e4-858d-c8d6cc464898.png)
Click on "Open an existing Android Studio Project"
And select the file called "build.gradle" in the project folder :smile: 
![image](https://cloud.githubusercontent.com/assets/8342509/5745390/5c95dea0-9c20-11e4-915c-fb653a7ba4c2.png)

**Comment by ChristianLoizou** [comment]
There is no build.gradle file in the rope i have downloaded, just gitattributes, gitignore and github files...

**Comment by ChristianLoizou** [comment]
repo*
